internal/service/controltower/control.go

```diff
@@ -23,31 +25,72 @@ import (
 	tfslices "github.com/hashicorp/terraform-provider-aws/internal/slices"
 	"github.com/hashicorp/terraform-provider-aws/internal/tfresource"
 	"github.com/hashicorp/terraform-provider-aws/internal/verify"
+	"github.com/hashicorp/terraform-provider-aws/names"
 )
 
 // @SDKResource("aws_controltower_control", name="Control")
 func resourceControl() *schema.Resource {
```

Comment:
Not sure if this is a best-effort resource but should there be a new test for `parameters` or an add-on to an existing?